pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[tool.poetry]
+name = "linguicator-predictor"
+version = "0.0.1"
+description = ""
+authors = ["Heitor Boschirolli <heitor.boschirolli@gmail.com>", "Danilo Ribeiro"]
```

Comment:
Let's change this to `Machine Grooving Studio <machine.grooving@gmail.com>`? What do you think?

If you agree please change the `mantainers` section as well.